data/wordlists/default_userpass_for_services_unhash.txt

```diff
@@ -1785,3 +1785,36 @@ AURORA$JIS$UTILITY$
 wlcsystem wlcsystem
 news 
 CPRM 
+root xc3511
+root vizxv
+root 888888
+root xmhdipc
+root juantech
+root 123456
+root 54321
+root 1111
+root 666666
+root klv123
+service service
+admin1 password
+666666 666666
+888888 888888
+ubnt ubnt
+root klv1234
+root Zte521
+root hi3518
+root jvbzd
+root anko
+root zlxx.
+root 7ujMko0vizxv
+root 7ujMko0admin
+root ikwb
+root dreambox
+root user
+root realtek
+root 00000000
+admin 1111111
+admin 54321
+admin 7ujMko0admin
+admin meinsm
+mother fucker
```

Comment:
What default creds are these for? Reminds me of the airOS worm, but is it really a service? Are any of these? (Some look to be.)